.github/workflows/update_iers.yml

```
@@ -0,0 +1,31 @@
+name: Auto-update IERS tables
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  update-iers:
+    name: Auto-update IERS tables
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Download latest IERS files
+      run: ./update_builtin_iers.sh
+      working-directory: astropy/utils/iers/data
+    - name: Commit changes
+      run: |
+        git config user.name 'Astrobot'
+        git config user.email 'tom@chi-squared.org'
+        git add astropy/utils/iers/data/Leap_Second.dat
+        git add astropy/utils/iers/data/eopc04_IAU2000.62-now
+        git commit -m "Update IERS Earth rotation and leap second tables"
+    - name: Create Pull Request
+      uses: peter-evans/create-pull-request@v3
+      with:
+        branch: update-iers-
```

Is the trailing `-` necessary?

```suggestion
        branch: update-iers
```